dev-tools/composer.json

```diff
@@ -7,6 +7,7 @@
         "jangregor/phpstan-prophecy": "^0.6",
         "mi-schi/phpmd-extension": "^4.3",
         "phpmd/phpmd": "^2.6",
+        "phpstan/phpstan": "0.12.17",
```

Comment:
Is the missing leading `^` intentional? Also, can we bump the version too `0.12.18`?

Reply:
Yes, I think it is.

Reply:
I updated to `0.12.18`.

The missing `^` is intentional: I don't want any PHPStan release to randomly break our builds with new errors but the `composer.lock` file is not versioned.